### PR TITLE
vagrant: bump net-next vagrant box version

### PR DESCRIPTION
**Pull request by borkmann**
[ Code fix for BPF datapath will come in separate pr. ]

Pull in bpf-next kernel fix for bpf_skb_adjust_room() helper [0].

  [0] https://lore.kernel.org/bpf/cover.1591108731.git.daniel@iogearbox.net/

Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>

### CODE REVIEW

**Comment by borkmann** [comment]
test-me-please

File: vagrant_box_defaults.rb
Comment (pchaigno) [inline vagrant_box_defaults.rb:8]: +3 already! :scream:

**Review by pchaigno — APPROVED** [review]
LGTM (as long as I can download the VM before this gets merged :)) )

File: vagrant_box_defaults.rb
Comment (borkmann) [reply vagrant_box_defaults.rb:8]: There was an issue in the packer builder on jenkins that @nebril recently fixed.

**Comment by borkmann** [comment]
retest-net-next

**Review by nebril — APPROVED** [review]

**Comment by borkmann** [comment]
retest-net-next

**Pull request merged**
